### DISCUSSION TITLE
howto change style on sibling node

### DISCUSSION THREAD

**Discussion by comes** [tailwindlabs/tailwindcss#7293] [Help] [answered]
Hej community,

I'm trying to find an equivalent to the following simple css code, but in class. something like

`<input class="group" type="checkbox" /><span class="group-checked:bg-green bg-red"> some text </span>`

```
input + span {
  @apply bg-red;
}

input:checked + span {
  @applu bg-green;
}
```

any ideas?

greetings, J

**Comment by sntran** [comment, marked as answer]
What you are looking for is `peer-*`: https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

In your example:

```html
<input class="peer" type="checkbox" />
<span class="peer-checked:bg-green bg-red"> some text </span>
```

**Reply by comes** [reply]
That is super need! I love it. Great
